### DISCUSSION TITLE
How to remove the background color due to hover in q-btn ?

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#11178] [General - Components / Directives / etc] [answered]
Consider this q-btn:

```vue
<q-btn
  flat
  round
  icon="account_circle"
  color="black"
  size="1rem"
>
```

Without hover the button will look like this:
![Screenshot from 2021-11-01 08-23-47](https://user-images.githubusercontent.com/76194919/139625453-cd95467f-f52d-49b1-a988-5aea9f6fb3ca.png)

With hover:
![Screenshot from 2021-11-01 08-23-59](https://user-images.githubusercontent.com/76194919/139625503-72c45452-deae-449b-86dd-7997631b63dd.png)

How can i remove the background shadow created due to hovering over the button ?

**Comment by hinogi** [comment]
A quick fix could be.
```css
.q-hoverable:hover > .q-focus-helper {
  visibility: hidden;
}
```

**Reply by ghost** [reply]
I tried it, it didn't work !

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/milospaunovic/pen/OJjvrxo](https://codepen.io/milospaunovic/pen/OJjvrxo)

**Reply by ghost** [reply]
That worked, thank you
